test/viewmodels/NationalInsuranceContributionsViewModelSpec.scala

```diff
@@ -36,16 +37,92 @@ class NationalInsuranceContributionsViewModelSpec extends SpecBase with TableDri
 
   private val nicUrl = nics.routes.Class2NICsController.onPageLoad(taxYear, NormalMode).url
 
+  private val nicCannotStartStatus: Option[JourneyNameAndStatus] = Some(
+    JourneyNameAndStatus(NationalInsuranceContributions, JourneyStatus.CannotStartYet))
+
+  private val nicNotStartedStatus: Option[JourneyNameAndStatus] = Some(JourneyNameAndStatus(NationalInsuranceContributions, JourneyStatus.NotStarted))
+
+  private val nicInProgressStatus: Option[JourneyNameAndStatus] = Some(JourneyNameAndStatus(NationalInsuranceContributions, JourneyStatus.InProgress))
+
+  private val nicCompleteStatus: Option[JourneyNameAndStatus] = Some(JourneyNameAndStatus(NationalInsuranceContributions, JourneyStatus.Completed))
+
   private val testScenarios = Table(
-    ("nationalInsuranceStatuses", "userAnswers", "expected"),
-    // No statuses, no answers, defaults to cannot start yet until the saving is implemented
-    (Nil, Nil, List(expectedRow(nicUrl, NationalInsuranceContributions, CannotStartYet)))
+    ("nationalInsuranceStatus", "businessStatuses", "expected"),
+    // If Business status does not contain completed adjustment status; NIC status should be CanNotStartYet
+    (
```

Comment:
if adjustments in businessStatuses are not ALL completed then the UI status will be CannotStartYet regardless of the saved NICStatus.

1. None / List(Model(List(Adj, InProgress))) -> CannotStartYet
2. NotStarted / List(Model(List(Adj, Completed)), Model(List(Adj, Completed)), Model(List(Adj, CSY))) -> CSY
3. InProgress / List(Model(List(Adj, Completed)), Model(List(Adj, InProgress))) -> CSY
4. Completed /  List(Model(List.empty)) -> CSY